packages/features/ee/billing/api/webhook/_invoice.paid.org.ts

```diff
@@ -36,6 +38,36 @@ async function handlePaymentReceivedForOnboarding({
   });
 }
 
+async function increaseRatelimitForOrganizationOwner(orgOwnerEmail: string) {
+  const user = await prisma.user.findUnique({
+    where: { email: orgOwnerEmail },
+    select: {
+      id: true,
+    },
+  });
+  if (!user) {
+    logger.error(`User not found for email: ${orgOwnerEmail}`);
+    return;
+  }
+
+  const { UNKEY_ROOT_KEY } = process.env;
+  if (!UNKEY_ROOT_KEY) {
+    logger.warn("UNKEY_ROOT_KEY is not set");
+    return;
+  }
+
+  const unkeyClient = new Unkey({ rootKey: UNKEY_ROOT_KEY });
+
+  await unkeyClient.ratelimit.setOverride({
```

Comment:
this coult silently fail, see: https://www.unkey.com/docs/libraries/ts/sdk/overview#checking-for-errors

```suggestion
  const res = await unkeyClient.ratelimit.setOverride({
  ...
  
  if (res.error){
    // log or throw or whatever
  }
```